src/Entity/Item.php

```diff
@@ -303,24 +302,6 @@ public function setStatements( StatementList $statements ) {
 	}
 
 	/**
-	 * @deprecated since 1.0, use getStatements()->toArray() instead.
-	 *
-	 * @return Statement[]
-	 */
-	public function getClaims() {
```

Comment:
This is unrelated to both commits in this pull request. Is this breaking change really necessary?